src/templates/base.html

```diff
@@ -25,6 +25,10 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800">
     <link rel="stylesheet" href="/static/css/styles.css">
     {% endblock %}
+		{% set view_args = request.view_args.copy() %}
+	{% for l in supported_languages %}
```

Comment:
This includes the current language. Is that ok for alternate hreflangs?

Reply:
Yes. The rules are : 
- You self reference (so if we are in /en/ we should point to /en/
- You reference tha alternate pages (so here we point to /ja/
- Bidirectional (A point to B, B should point to A)